plugins/action_modify_record/src/modifyRecord.ts

```diff
@@ -78,6 +87,70 @@ const modifyRecord: ActionPluginType = async ({ parameters, applicationData, DBC
   }
 }
 
+// When we have multiple records, we recursively call "modifyRecord" on each
+// one, and collect all outputs
+const updateMultipleRecords: ActionPluginType = async ({
+  parameters,
+  applicationData,
+  DBConnect,
+}) => {
+  let status = ActionQueueStatus.Success
+  const errors: string[] = []
+  const {
+    tableName,
+    matchField,
+    matchValue,
```

Comment:
It seems a little bit weird to update multiple records matched by the same value

Reply:
Yeah, again -- not really likely in practice, but don't see the need to actively prevent it either.